core/adapters/bridge.go

```diff
@@ -34,10 +35,29 @@ func (ba *Bridge) Perform(input models.RunInput, store *store.Store) models.RunO
 	} else if input.Status().PendingBridge() {
 		return models.NewRunOutputInProgress(input.Data())
 	}
-	return ba.handleNewRun(input, store.Config.BridgeResponseURL())
+	meta := getMeta(store, input.JobRunID())
+	return ba.handleNewRun(input, meta, store.Config.BridgeResponseURL())
 }
 
-func (ba *Bridge) handleNewRun(input models.RunInput, bridgeResponseURL *url.URL) models.RunOutput {
+func getMeta(store *store.Store, jobRunID *models.ID) *models.JSON {
+	jobRun, err := store.ORM.FindJobRun(jobRunID)
+	if err == nil && jobRun.ID != nil && jobRun.RunRequest.TxHash != nil && jobRun.RunRequest.BlockHash != nil {
```

Comment:
To ease the cognitive burden on other Go developers (specifically, the one created by having to scan over so many `if err != nil` blocks), it's always recommended to error out first, and then check other conditions.  Additionally, it's always recommended to make the "happy path" as unindented as possible.  So I'd probably rearrange the logic like this:
```go
func getMeta(store *store.Store, jobRunID *models.ID) *models.JSON {
	jobRun, err := store.ORM.FindJobRun(jobRunID)
    if err != nil {
        return nil
    } else if jobRun.ID == nil || jobRun.RunRequest.TxHash == nil || jobRun.RunRequest.BlockHash != nil {
        return nil
    }
    // ...
```